docs/knowledgebase/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "knowledgebase",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "Unlicense",
+  "scripts": {
+    "update-doc-versions": "find . -type f -name \"*.md\" -print0 | xargs -0 sed -i -e 's/substrate.dev\\/rustdocs\\/'\"$OLD_VERSION\"'/substrate.dev\\/rustdocs\\/'\"$NEW_VERSION\"'/g'"
+  }
+}
```

Comment:
Can you say a  little about this package.json file?

Why is it nested inside the repo? Is this update script supposed to work alongside docusaurus's own versioning? https://docusaurus.io/docs/en/versioning

Reply:
This could easily be moved elsewhere, I was just looking for a place to encapsulate the logic of updating all the Rustdocs links when a new version of Substrate is released. I would say this is intended to be used _as opposed to_ Docusaurus's own versioning. Here's how this little script can be used to update the links: `OLD_VERSION=v2.0.0-rc3 NEW_VERSION=v2.0.0-rc4 yarn update-doc-versions`